src/slic3r/GUI/ConfigManipulation.cpp

```diff
@@ -282,7 +283,7 @@ void ConfigManipulation::update_print_fff_config(DynamicPrintConfig* config, con
            ! config->opt_bool("overhang_reverse") &&
             config->opt_enum<TimelapseType>("timelapse_type") == TimelapseType::tlTraditional))
     {
-        wxString msg_text = _(L("Spiral mode only works when wall loops is 1, support is disabled, top shell layers is 0, sparse infill density is 0 and timelapse type is traditional."));
+        wxString msg_text = _(L("Spiral mode only works when wall loops is 1, alternate extra wall is disabled, support is disabled, top shell layers is 0, sparse infill density is 0 and timelapse type is traditional."));
```

Comment:
I'm thinking about this.
I think we should do otherway around, when spiral vase is used we should skip extra wall in the code(PerimeterGenerator class) automatically.
PerimeterGenerator has been already checking spiral mode(by using m_spiral_vase), so we should just avoid the loop+1 logic when m_spiral_vase is true.

Let me know what you think.
```suggestion
        wxString msg_text = _(L("Spiral mode only works when wall loops is 1, alternate extra wall is disabled, support is disabled, top shell layers is 0, sparse infill density is 0 and timelapse type is traditional."));
```

Reply:
no preference from my side - either works ;) ill update the pr soon

Reply:
Done. Also I removed the ensure vertical shell thickness check in vase mode - this is not needed for sure and it was clashing with the alternate extra wall pop up that I just implemented in the change above ;)